packages/flutter_test/lib/src/binding.dart

```diff
@@ -894,6 +894,9 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
           stack: stack,
           context: ErrorDescription('running a test (but after the test had completed)'),
           library: 'Flutter test framework',
+          informationCollector: [
+            ErrorHint("This does not cause test failures, but may indicate your code has bugs."),
```

Comment:
This is basically what is already said at the top, that this happened AFTER the test completed.

Reply:
I think so. The problem is, the original text does not mention this will *not* make any test fail. For example, if something throws in tearDown or even tearDownAll, it is *after* the test but it will make `flutter test` report failure.